spec/auth/armor_spec.rb

```diff
@@ -1,9 +1,44 @@
+require "auth_armor"
+
 RSpec.describe Auth::Armor do
   it "has a version number" do
     expect(Auth::Armor::VERSION).not_to be nil
   end
 
-  it "does something useful" do
-    expect(false).to eq(true)
+  #it "does something useful" do
+  #  expect(false).to eq(true)
+  #end
+
+  describe connect do
+  it "Connects successfully" do
+    expect{200}.to output{"Connection successfull"}.to_stdout
+  end  
+  it "invalid connection" do 
+    expect{400}.to output{"Error"}.to_stdout
+  end
+  it "fails" do
```

Comment:
Give a more descriptive test spec that's related to the test itself. For example, it returns a 400 error